.bazelrc

```diff
@@ -1,2 +1,4 @@
 # build config
 build --compilation_mode=opt
+build --action_env=PATH
```

Comment:
@pcmoritz I think by passing `PATH` to bazel, #4334 is not needed any more. But I don't know how to verify that.

Reply:
It is still needed, otherwise we can't guarantee which python will be picked up if there are multiple ones.

Reply:
ok, just noticed that the `build.sh` script takes a `-p` option.